summary/usergroup_test.go

```diff
@@ -228,12 +229,25 @@ func testBadIds(err error, a byColumnAdder, out *os.File, outPath string) {
 	So(output, ShouldContainSubstring, "id999999999")
 }
 
-func checkFileIsSorted(path string) bool {
-	cmd := exec.Command("sort", "-C", path)
+func checkDGUTAFileIsSorted(path string) bool {
```

Comment:
From the diff it looks like you added a few *IsSorted funcs, but didn't use them?

Reply:
They're used in other test files; I'll move them to the appropiate files.